.htaccess

```diff
@@ -6,7 +6,11 @@ Update this file if you have an entry that you are moving around.
 Don't update this file if you are adding a new entry.
 {% endcomment %}
 
-redirect 301 /doc/tutorial.html {{ BASE_PATH }}/tutorial/basic/
+redirect 301 /doc/tutorial.html {{ BASE_PATH }}/doc/tutorial/basic/
+redirect 301 /tutorial/basic/ {{ BASE_PATH }}/doc/tutorial/basic/
+redirect 301 /tutorial/ {{ BASE_PATH }}/doc/tutorial/
+redirect 410 /tutorial/boost/
+redirect 410 /tutorial/firstapp/
```

Comment:
I must note that this `.htaccess` is inconsistent, some parts of it use this `BASE_PATH`, others don't.